### ISSUE TITLE
Problem importing package starting at v1.2.25

### ISSUE THREAD

**Issue by jasonbarry** [devforth/painterro#123]
**Describe the bug**
I'm getting a webpack error when I try to call `Painterro()`: 
```
TypeError: painterro__WEBPACK_IMPORTED_MODULE_1___default() is not a function. (In 
'painterro__WEBPACK_IMPORTED_MODULE_1___default()(configOptions)', 
'painterro__WEBPACK_IMPORTED_MODULE_1___default()' is an instance of Object)
```

**Painterro version**
v1.2.25

**To Reproduce**
Here's my source

```js
import Painterro from 'painterro'

const configOptions = {
  activeColor: '#ff0000',
}

export default function Annotate() {
  const show = () => {
    Painterro(configOptions).show()
  }

  return (
    <button onClick={show}>Annotate</button>
  )
}
```

The error occurs when I click the button. With version v1.2.24, it works as expected.

**Comment by ivictbor** [comment]
Thanks for the report. Yep I update some webpack packages, will try to track the issue

**Comment by ivictbor** [comment]
@jasonbarry after webpack update my exported function which created Painterro class stated to be wrapped in object by webpack/babel.

For now I am not sure what happed but obviously temporary resolution for commonjs2 imports (npm installs) can change

```
import Painterro from 'painterro'
```

To 

```
import { Painterro } from 'painterro'
```

And it will work. Var version (script tag should work as it was before)

I will spend some time trying to understand why webpack stopped export function and if I will not able to do it, we will release 1.3.x minor update which will break backward compatibility.

**Comment by ivictbor** [comment]
@jasonbarry I prepared minimal example and created an issue on webpack to understand what happened https://github.com/webpack/webpack/issues/12675

**Comment by ivictbor** [comment]
@jasonbarry fixed in 1.2.31

The good news that now we have updated webpack & babel to the latest version

**Comment by jasonbarry** [comment]
Confirmed that it works in 1.2.31! Thanks for fixing.
